README.md

```diff
@@ -1,84 +1,66 @@
- - GHC 9.0.1
- - GHC HEAD.
+- [Travis-CI](https://travis-ci.org/)
+- [GitHub Actions](https://docs.github.com/en/actions)
 
-Each GHC version is provided in a separate `ghc-<version>` `.deb` package installing into `/opt/ghc/<version>` (thus allowing to be installed at the same time if needed) published in a [PPA](https://launchpad.net/~hvr/+archive/ghc). The easiest way to "activate" a particular GHC version is to prepend its `bin`-folder to the `$PATH` environment variable (see example in next section).
+Features include:
 
-Note: For actually enabling continuous integration for a GitHub hosted project, see section [Getting Started](http://about.travis-ci.org/docs/user/getting-started/) in [Travis-CI](https://travis-ci.org/)'s online documentation.
+- Multiple [GHC](http://haskell.org/ghc) support
+- Building with specific constraints
+- Dependency caching
+- cabal and cabal.project support (see [Nix-style local builds documentation](https://cabal.readthedocs.io/en/latest/nix-local-build-overview.html))
+- Runs tests and builds benchmarks
+- Support for Haskell build tools, including [Haddock](https://hackage.haskell.org/package/haddock), [GHCJS](https://github.com/ghcjs/ghcjs) & [HLint](https://hackage.haskell.org/package/hlint)
+- doctest runner integration using either [doctest](https://hackage.haskell.org/package/doctest) or [cabal-docspec](https://github.com/phadej/cabal-extras/blob/master/cabal-docspec/MANUAL.md).
 
-### Quick-start instructions
+Usage
+---
 
-* Step 1: Clone and install this project in/from any directory
+See `haskell-ci --help` for details.
 
-    ```bash
-    $ git clone https://github.com/haskell-CI/haskell-ci.git
-    $ cd haskell-ci
-    $ cabal new-install haskell-ci:exe:haskell-ci
-    ```
+1. Install `haskell-ci`
 
-  or
+```bash
+cabal install haskell-ci
+```
 
-    ```bash
-    cabal new-install haskell-ci
-    ```
+2. Add a `tested-with` line to your .cabal file (e.g. `tested-with: GHC == 8.10.4 || == 9.0.1`)
 
-    *Note:* currently (2019-02-12) released version of `haskell-ci` installs
-    an executable named `make-travis-yml`
+3. Create a configuration file. A good starting point for this is the `dump-config` command eg
 
-* Step 2: Change directories to your project:
+``` shell
+haskell-ci dump-config > haskell-ci.config
+```
 
-    ```bash
-    $ cd path/to/your-project
-    ```
+4. Run the `github` or `travis` commands as appropriate to your workflow. eg
 
-* Step 3: Edit your project's `*.cabal` file to add a `Tested-With` line, such as this one:
+``` shell
+haskell-ci github '--config=haskell-ci.config' 'cabal.project' --output .github/workflows/haskell-ci.yml
+```
 
-    ```bash
-    $ cat your-project.cabal
-    ...
-    Tested-With: GHC ==8.6.3 || ==8.4.4 || ==8.2.2
-    ...
-    ```
-    
-    Add as many or as few GHC versions to test as you want.
+``` shell
+haskell-ci travis '--config=haskell-ci.config' 'cabal.project' --output .travis.yml
+```
```

Comment:
Is there a particular reason for specifying the default output paths via `--output`? I've never felt the need to use that option.